packages/core/content-type-builder/admin/src/components/ComponentIconPicker/index.js

```diff
@@ -16,23 +16,14 @@ import Cell from './Cell';
 
 const CELL_WIDTH = 44;
 
-const ComponentIconPicker = ({ error, isCreating, intlLabel, name, onChange, value }) => {
-  const { allIcons, allComponentsIconAlreadyTaken } = useDataManager();
+const ComponentIconPicker = ({ error, intlLabel, name, onChange, value }) => {
+  const { allIcons } = useDataManager();
   const { formatMessage } = useIntl();
-  const [originalIcon] = useState(value);
-  const initialIcons = allIcons.filter(ico => {
-    if (isCreating) {
-      return !allComponentsIconAlreadyTaken.includes(ico);
-    }
-
-    // Edition
-    return !allComponentsIconAlreadyTaken.filter(icon => icon !== originalIcon).includes(ico);
-  });
```

Comment:
Does this mean that several components can have the same icon? I guess it's on purpose, right?

Reply:
Yes, they could previously but users found it confusing and we decided to allow duplicates.